Internal/Debug App/Sources/Network/Networking.swift

```diff
@@ -96,11 +96,10 @@ class Networking {
                         
         msg += "Headers:\n\(request.allHTTPHeaderFields ?? [:])\n"
                 
-        if let body = body {
+        if let body = body,
+           let bodyJson = try? JSONSerialization.jsonObject(with: body, options: .allowFragments){
             request.httpBody = body
-            
-            let bodyJson = try? JSONSerialization.jsonObject(with: body, options: .allowFragments)
-            msg += "Body:\n\(bodyJson ?? [:])\n"
```

Comment:
This was written this way as to always have console log and know what's happening, even if the body was `nil`.

Reply:
OK, reversed